atomic_reactor/utils/operator.py

```diff
@@ -734,6 +734,30 @@ def _replace_pullspecs_not_in_annotations(self, obj, k_or_i, replacements):
             # in replacement_pullspecs and no replacement will be done
             self._replace_unnamed_pullspec(obj, k_or_i, replacements)
 
+    def modifications_append(self, append_mods):
+        if not append_mods:
+            return
+        for mod in append_mods:
+            for key in append_mods[mod]:
+                item = append_mods[mod][key]
+                if isinstance(item, list):
+                    self.data[mod].setdefault(key, []).extend(item)
+                else:
+                    #  Raise something - we're expecting a list
```

Comment:
:eyes:

Reply:
@MartinBasti; Yes... What should I raise here?

Reply:
`RuntimeError`